lightkurve/convenience.py

```diff
@@ -9,22 +9,22 @@
 
 
 def cdpp(flux, **kwargs):
```

Comment:
Should we have something that is a method (verb) also be allowed to be a property (noun)? I think this is more confusing? It's estimating every time.

Reply:
Agreed!  Will change.